docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
     network_mode: host
 
   influxdb_v2:
-    image: quay.io/influxdb/influxdb:2.0.0-beta
+    image: quay.io/influxdb/influx:nightly
```

Comment:
Why was an image change to `nightly`?

Reply:
Mistake, there should be quay.io/influxdb/influxdb:2.0.0-beta.